components/atoms/button/components/button/button.js

```diff
@@ -3,7 +3,7 @@
  */
 
 // React
-import { forwardRef } from 'react'
+import React, { forwardRef } from 'react'
```

Comment:
@majid-amiri Do we need to explicitly import react now ?

Reply:
No. I will revert it.